CODEOWNERS

```diff
@@ -1,37 +1,194 @@
+#   system: Make, Shell, Docker.
+# - @cilium/cli:
+#   Provide user experience feedback on changes to Command-Line Interfaces.
+#   These owners are a stand-in for the user community to bring a user
+#   perspective to the review process. Consider how information is presented,
+#   consistency of flags and options.
+# - @cilium/ci-structure:
+#   Guidance around the best use of Cilium project continuous integration and
+#   testing infrastructure, including GitHub actions, VM helpers, testing
+#   frameworks, etc.
+# - @cilium/contributing:
+#   Encourage practices that ensure an inclusive contributor community. Review
+#   tooling and scripts used by contributors.
+# - @cilium/docs-structure:
+#   Ensure the consistency and layout of documentation. General feedback on the
+#   use of Sphinx, how to communicate content clearly to the community. This
+#   code owner is not expected to validate the technical correctness of
+#   submissions. Correctness is typically handled by another code owner group
+#   which is also assigned to any given piece of documentation.
+# - @cilium/github-sec:
+#   Responsible for maintaining the security of repositories in the Cilium
+#   project by maintaining best practices for workflow usage, for instance
+#   preventing malicious use of GitHub actions.
+# - @cilium/helm:
+#   Provide input on the way that Helm can be used to configure features. These
+#   owners are a stand-in for the user community to bring a user perspective to
+#   the review process. Ensure that Helm changes are defined in manners that
+#   will be forward-compatible for upgrade and follow best practices for
+#   deployment (for example, being GitOps-friendly).
+# - @cilium/metrics:
+#   Provide recommendations about the types, names and labels for metrics to
+#   follow best practices. This includes considering the cardinality impact of
+#   metrics being added or extended.
+# - @cilium/tophat:
+#   Top Hat duties rotate between the committer group from week to week, and
+#   they may assist in maintenance, triage and backporting duties across
+#   different Cilium repositories. Catch-all for code not otherwise owned by a
+#   team.
+# - @cilium/vendor:
+#   Review vendor updates for software dependencies to check for any potential
+#   upstream breakages / incompatibilities. Discourage the use of unofficial
+#   forks of upstream libraries if they are actively maintained.
+#
+# Repository Owners
+# +++++++++++++++++
+#
+# The following code owners are responsible for a range of general feedback for
+# specific repositories:
+#
+# - @cilium/hubble-ui:
+#   Maintains the Hubble UI graphical interface.
+# - @cilium/sig-agent:
+#   Provides Cilium (agent) general Go review. Internal architecture, core data
+#   structures and daemon startup.
+# - @cilium/tetragon:
+#   Review of all Tetragon code, both for Go and C (for eBPF).
+#
+# These teams are also available to contact the maintainers of specific
```

Comment:
I'm not sure I understand "available to contact" - does this mean, if you're looking for advice about these components you should use these teams to contact the relevant maintainers?

Reply:
That was my thinking yeah. If we're happy with stating this in the docs, maybe I can just wordsmith this a little more to make it clearer to the reader.

Reply:
One of the things I've struggled with a little bit in defining these docs is that we do not generally expect that the "maintainers" provide all of the code review. "Maintainers" typically have the admin rights and will often set processes, assist with making architectural decisions and so on. But the actual review of individual contributions is mostly delegated to specific groups of committers. So for instance @cilium/tetragon is a wider group than @cilium/tetragon-maintainers , and it's the tetragon group that reviews all code in the tetragon repo. The maintainers will typically be part of one or more other codeowners as well, so they likely will be providing additional review, but not wearing their "admin" / "maintainer" hat. So in the codeowners files, you will likely not see @cilium/tetragon-maintainers assigned to review specific lines of code.

Reply:
Here's a new proposed wording that I'll push shortly:

> The teams above are responsible for reviewing the majority of contributions                                                             
> to the corresponding repositories. Additionally, there are "maintainer" teams                                                           
> listed below which may not be responsible for overall code review for a                                                                 
> repository, but they have administrator access to the repositories and so                                                               
> they can assist with configuring GitHub repository settings, secrets, and                                                               
> related processes. For the full codeowners for individual repositories, see                                                             
> the CODEOWNERS file in the corresponding repository.